Dockerfile

```diff
@@ -14,17 +39,58 @@ COPY requirements /src/requirements
 COPY extend-environment.sh /etc/profile.d/extend-environment.sh
 
 # Make sure pip is installed
-ADD https://bootstrap.pypa.io/get-pip.py /src/get-pip.py
-RUN python2.7 /src/get-pip.py 
+RUN pip2 install -U pip setuptools
 RUN python3.6 -m ensurepip
+RUN pip3.6 install -U pip setuptools
 
-# Make sure pip is up to date
-RUN pip2.7 install -U pip
-RUN pip3.6 install -U pip
-
-# Install our requirements
+# Install our python requirements
 RUN pip2.7 install -r /src/requirements/local.txt
 RUN pip3.6 install -r /src/requirements/local.txt
 
-# Make the .env available in the bash shell
-RUN echo 'source /src/cfgov-refresh/.env' >> ~/.bashrc
+ENV EXTRA_MODWSGI_ARGS "--reload-on-changes"
+
+FROM cfgov-dev as cfgov-build
+RUN mkdir /src/cfgov-refresh
+WORKDIR /src/cfgov-refresh
+
+COPY scripts scripts
+COPY frontend.sh .
+COPY package.json .
+COPY code.json .
+COPY yarn.lock .
+COPY babel.config.js .
+COPY gulpfile.js .
+COPY gulp gulp
+COPY config config
+COPY jest.config.js .
+COPY cfgov/ ./cfgov/
+COPY cfgov-fonts-master.zip /tmp/fonts.zip
```

Comment:
Isn't this going to make it so this image is only buildable by someone (or some service) with access to our GHE.  Could we just copy the checkout(s) right into `static.in`, that way the build would still work if that directory was empty?

Reply:
right now, collectstatic fails if those files aren't present--  so I think 

> only buildable by someone (or some service) with access to our GHE

describes the status-quo?